pkg/webhook/risingwave_validating_webhook.go

```diff
@@ -207,7 +230,7 @@ func (v *RisingWaveValidatingWebhook) validateCreate(ctx context.Context, obj *r
 
 	// Validate the global spec.
 	//   * If global replicas of any component is larger than 1, then the image in global must not be empty.
-	fieldErrs = append(fieldErrs, v.validateGlobal(field.NewPath("spec", "global"), &obj.Spec.Global)...)
+	fieldErrs = append(fieldErrs, v.validateGlobal(field.NewPath("spec", "global"), &obj.Spec.Global, *obj.Spec.EnableOpenKruise)...)
```

Comment:
Use `pointer.BoolDeref` in package `k8s.io/utils/pointer` to avoid nil reference.

```suggestion
	fieldErrs = append(fieldErrs, v.validateGlobal(field.NewPath("spec", "global"), &obj.Spec.Global, pointer.BoolDeref(obj.Spec.EnableOpenKruise))...)
```

Reply:
adjusted in the latest commit

Reply:
It seems this one is not fixed yet.

Reply:
hey Shunjie apologies, but I have not pushed the latest commit with this change as well as the additional test cases and validation. Will get it out by today.